cmd/ingress/ingress.go

```diff
@@ -139,6 +139,7 @@ the apisix cluster and others are created`,
 	cmd.PersistentFlags().StringVar(&cfg.LogLevel, "log-level", "info", "error log level")
 	cmd.PersistentFlags().StringVar(&cfg.LogOutput, "log-output", "stderr", "error log output file")
 	cmd.PersistentFlags().StringVar(&cfg.HTTPListen, "http-listen", ":8080", "the HTTP Server listen address")
+	cmd.PersistentFlags().StringVar(&cfg.HTTPSListen, "https-listen", ":443", "the HTTPS Server listen address")
```

Comment:
We'd better not to use the well-known `443` port here:

1. Listen on `443` requiring root permission
2. it's not consistent with the HTTP listen (`8080`).

Reply:
Or use 8443 instead.